addons/blocks2image/addon.json

```
@@ -1,11 +1,15 @@
 {
   "name": "Save blocks as image",
-  "description": "Right click the code area to export blocks as SVG/PNG images.",
+  "description": "Right-click on the code area or a stack of blocks to copy it to the clipboard or save it as a PNG or SVG file.",
```

Consistency with other addon descriptions.
```suggestion
  "description": "Right click on the code area or a stack of blocks to copy it to the clipboard or save it as a PNG or SVG file.",
```

---

The manifest docs say to use `scripts` instead of `stacks of blocks`, but eh... I think "stack of blocks" is more clear here, and 3 other addons already omit this rule.